scripts/nice2/helpers.sh

```diff
@@ -103,9 +103,20 @@ function waitForWidgetServer() {
   echo "widget-server is ready"
 }
 
-function emptyDB() {
+function setupDB() {
+  if doesTemplateExist;
```

Comment:
question: should we really support both versions?

Reply:
For backwards-compatibility reasons I would not.. Not sure if we really want to maintain the 3 tests we have on > 3.4

Reply:
I've removed the creation of the dump, but I'd suggest we keep the reading of the dump for now. It's an easy way to run cypress on a specific database state without having to restore it yourself, and makes the transition to a template database seamless for anyone that already has cypress setup the old way.

(It's also a neat fallback in case I've included any breaking bugs ... :) )